R/mod_ParticipantDomain_Server.R

```diff
@@ -3,50 +3,63 @@
 #' Display a specific (named) table, or a placeholder.
 #'
 #' @inheritParams shared-params
-#' @returns A [shiny::reactive()] with the index(es) of any rows selected by the
-#'   user.
+#' @returns A [shiny::reactive()] with the SubjectID for each row that is
+#'   selected. Currently not useful but will be converted to row numbers when we
+#'   need it for deeper dives.
 #' @keywords internal
 mod_ParticipantDomain_Server <- function(id, rctv_lData, rctv_strName) {
   moduleServer(id, function(input, output, session) {
-    valid_lData <- reactive({
+    rctv_lDataIsValid <- reactive({
       length(rctv_lData()) > 0
     })
-    valid_selection <- reactive({
-      valid_lData() &&
+    rctv_selectionIsValid <- reactive({
+      rctv_lDataIsValid() &&
         length(rctv_strName()) &&
         rctv_strName() %in% names(rctv_lData())
     })
-    output$details <- renderUI({
-      if (valid_selection()) {
-        return(DT::DTOutput(session$ns("table"), fill = FALSE))
+    rctv_tblData <- reactive({
+      if (rctv_selectionIsValid()) {
+        df <- rctv_lData()[[rctv_strName()]]
+        return(df)
       }
-      return(out_Placeholder(
-        c(
-          if (!valid_lData()) "participant",
-          "domain"
-        )
-      ))
+      dplyr::tibble(SubjectID = character()) # nocov
     })
-    output$title <- renderText({
-      if (!valid_selection()) {
-        return("Participant Domain")
+
+    rctv_gtObject <- reactive({
+      if (rctv_selectionIsValid()) {
+        df <- rctv_tblData()
+        gtObj <- gt::gt(df) %>%
+          out_gtInteractive(selection_mode = "multiple") %>%
```

Comment:
when i select multiple, the app crashes, like below- 
![image](https://github.com/user-attachments/assets/1d790144-37f3-4945-b08a-9c630cbb9e85)

Reply:
Fixed in latest push.